types/webpack/index.d.ts

```diff
@@ -646,6 +648,118 @@ declare namespace webpack {
+             */
+            inline?: boolean;
+            /**
+             * When lazy is enabled, the dev-server will only compile the bundle when it gets requested.
+             * This means that webpack will not watch any file changes.
+             */
+            lazy?: boolean;
+            /**
+             * With noInfo enabled, messages like the webpack bundle information that is shown
+             * when starting up and after each save,will be hidden.
+             * Errors and warnings will still be shown.
+             */
+            noInfo?: boolean;
+            /** When open is enabled, the dev server will open the browser. */
+            open?: boolean;
+            /** Specify a page to navigate to when opening the browser. */
+            openPage?: string;
+            /** Shows a full-screen overlay in the browser when there are compiler errors or warnings. Disabled by default. */
+            overlay?: boolean | object;
+            /** When used via the CLI, a path to an SSL .pfx file. If used in options, it should be the bytestream of the .pfx file. */
+            pfx?: string;
+            /** The passphrase to a SSL PFX file. */
+            pfxPassphrase?: string;
+            /** Specify a port number to listen for requests on. */
+            port?: number;
+            /**
+             * Proxying some URLs can be useful when you have a separate API backend development server
+             * and you want to send API requests on the same domain.
+             */
+            proxy?: object;
+            /**
+             * When using inline mode and you're proxying dev-server,
+             * the inline client script does not always know where to connect to.
+             * It will try to guess the URL of the server based on window.location, but if that fails you'll need to use this.
+             */
+            public?: string;
+            /** The bundled files will be available in the browser under this path. */
+            publicPath?: string;
+            /**
+             * With quiet enabled, nothing except the initial startup information will be written to the console.
+             * This also means that errors or warnings from webpack are not visible.
+             */
+            quiet?: boolean;
+            /** @deprecated Here you can access the Express app object and add your own custom middleware to it. */
+            setup?: (app: Application) => void;
+            /** The Unix socket to listen to (instead of a host). */
+            socket?: string;
+            /** It is possible to configure advanced options for serving static files from contentBase. */
+            staticOptions?: object;
+            /**
+             * This option lets you precisely control what bundle information gets displayed.
+             * This can be a nice middle ground if you want some bundle information, but not all of it.
+             */
+            stats?: string | object;
```

Comment:
Instead of object use the Stats interface, which is already created. 

Options.Stats